CMakeLists.txt

```diff
@@ -126,12 +125,20 @@ if(reporting)
   )
 endif()
 
+if(coverage)
+  if(CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
+    message(WARNING "Code coverage on Windows is not supported, ignoring 'coverage' flag")
+  else()
+    include(CodeCoverage)
```

Comment:
I think it would be good to move this include into `RippledCov.cmake`.

Reply:
Thank you, done :+1: